relays/client-substrate/src/lib.rs

```diff
@@ -57,10 +57,15 @@ pub use bp_runtime::{
 /// Substrate-over-websocket connection params.
 #[derive(Debug, Clone)]
 pub struct ConnectionParams {
+	/// Websocket endpoint URL. Overrides all other URL components (`host`, `port`, `path` and
+	/// `secure`).
+	pub uri: Option<String>,
 	/// Websocket server host name.
 	pub host: String,
 	/// Websocket server TCP port.
 	pub port: u16,
+	/// Websocket endpoint path at server.
+	pub path: Option<String>,
```

Comment:
Should we mark `host`, `port`, `path` and `secure` as deprecated in order to not forget about them ?

Reply:
Ok, makes sense. 

> I could add a suffix to associated CLI arguments: "Deprecated: use uri instead" as a compromise. WDYT?

I don't know if it's necessary. We can leave it as it is. At most we can add a TODO comment, just to make sure we don't forget about it.